Game/lv/Game.po

```diff
@@ -669,7 +669,7 @@ msgstr "Loga režīms"
 #: /Game/Resources/Localisation/ST_Menus.ST_Menus
 msgctxt "Menus,GameTag"
 msgid "ACT II"
-msgstr ""
+msgstr "DAĻA II"
```

Comment:
Thanks for this! Do you think this could stay as ACT II in English, since it's the brand? I'm trying to keep this the same in languages that are latin-based, so only Chinese, Japanese and Russian (etc) have it translated. What do you think?

Reply:
Yeah, better keep the english variant. I wasn't really sure how to translate that so I picked my variant. Btw, how to quickly but privately get in touch with you? I've started learning how to make games in Source Engine so I may need assistance. I watched your tutorials on Youtube. Now trying to figure out how everything works...

Reply:
Discord is the easiest way!